src/main/java/at/hannibal2/skyhanni/utils/NEUItems.kt

```diff
@@ -280,13 +285,24 @@ object NEUItems {
 
     fun allNeuRepoItems(): Map<String, JsonObject> = NotEnoughUpdates.INSTANCE.manager.itemInformation
 
+    fun getInternalNamesForItemId(item: Item): List<NEUInternalName> {
+        if (itemIdCache.contains(item)) {
+            return itemIdCache[item]!!
+        }
```

Comment:
Use a more Kotlin-like apprach
```suggestion
        itemIdCache[item]?.let {
            return it
        }
```